devel/tldr-pages/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        tldr-pages tldr 5a38d5f9f7dff0111fa756fdbbfc1546aa6905c7
```

Comment:
you will need to add a `version` field that sets a reasonable value, we do not want/can use a git hash for that as it doesn't necessarily "increase" as a usual version number. Other ports use, for example, the data of the commit as the `version` (of course in the form `YYYYMMDD`)